plugins/capistrano/capistrano.plugin.zsh

```diff
@@ -0,0 +1,11 @@
+# Added `shipit` because `cap` is a reserved word. `cap` completion doesn't work.
```

Comment:
@apjanke thanks :beers: so is there a way to make the completion to work using `cap`?